WorkingAtBabylon.md

```diff
@@ -0,0 +1,90 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+Working in the iOS team
+==================================
+
+TODO
+
+This is more or less modelled after https://github.com/codeOfRobin/Questions-to-ask-your-interviewer-iOS
+
+### Where do we store our code?
+We use Github. We have a central repo that hosts the main and white label apps, and a [few Open Source projects](https://github.com/search?q=topic%3Aios+org%3ABabylonpartners+fork%3Atrue) that we created and rely on.
+
+## Continuous Integration
+### How many apps/targets/schemes?
+### How big is your Fastfile?
+It's around 250 lines of code at the moment, with about 15-20 lanes for beta testing, daily and custom builds, etc.
+
+### Does it have several environment references?
+### Can you dry run a lane on your machine right now?
+Yes.
+
+### Codesigning?
+  
+* What architectural patterns are you using to make development easier or more consistent? What problems with your current approach have you come across after settling on one (if you have one)?
+
+## Team/People
+### comms - slack?
+Slack, as a non-trivial part of the team is remote. Zoom for meetings of more than 2-3 people.
+
+### Do people ever take sick days to get work done?
+No.
+
+### When was the last time someone worked a late night/weekend in order to meet a deadline?
+TODO
+
+### How many meetings do you have in a day?
+It depend on the squad you’re assigned. For Enrolment and Integrity I counted 18 in a typical sprint, so that averages to ~2 a day. Note that quite a few of them are optional.
+
+Interviews, sometimes, can take up some time. It's usually, at maximum, one hour per week.
+
+### Does your calendar look like a losing game of tetris? Do you have large continuous uninterrupted hours?
+It depends a lot on the squad. For Enrolment and Integrity, for example, most meetings are on the first Monday and Tuesday of the sprint. For Triage and Health Management this is not the case and are more spread out.
+
+### Are there any private spaces in your office?
+There are very few at the moment.
+
+### Is there a gym/field/court close to your office for physical activity?
+There is a commercial gym close by, a park, and a barebones gym in the office. There’s also free yoga twice a week and a football club
+
+### Do people feel comfortable calling attention to systemic issues/heavily “band-aid”-ed systems?
+Yes, we have anonymous company level Q&A and peakon surveys. [this is my experience].
+
+### Are any of your engineering managers mobile engineers? Do mobile devs feel like they have to learn backend dev to "prove themselves" / feel like they have a seat at the table?
+iOS engineers have to know iOS and general programming practices, and that’s it.
```

Comment:
The first part of the question is not addressed and regarding the second bit I think we should say that every engineer can and should make proposals to improve our processes (thus, everyone has a seat at the table).

Reply:
Yep. Worth mentioning that are current line manager and the level above him are both (or were at some point) heavily involved in mobile development